src/search.ts

```diff
@@ -44,6 +44,21 @@ export interface SearchFilter {
     logicalOperator: 'AND' | 'OR';
 }
 
+export interface SearchCapabilityStatement {
+    [resourceType: string]: SearchCapabilities;
+}
+
+export interface SearchCapabilities {
+    searchParam: {
+        name: string;
+        definition: string;

```

Comment:
Based on [this](https://www.hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.rest.resource.searchParam.definition), the cardinality is `0..1`

Looks like `definition` and `documentation` can be optional?

Reply:
true. I changed both to be optional. Our implementation will always include those fields though, but this makes the interface more flexible for others.